File: C#/EntityFramework/Models/Person.cs
namespace Models;

public class Person
{
    public Guid Id { get; set; }
    public string? Name { get; set; }
    public string? Address { get; set; }
    public DateTime BirthDate { get; set; }

    public override string ToString()
    {
        return $"Person {{ Id: {Id}, Name: {Name}, Address: {Address}, BirthDate: {BirthDate} }}";
    }
}

File: FSharp/EntityFramework/Models/Person.fs
namespace Models

open System
open System.ComponentModel.DataAnnotations

type Person () =
  [<Key>]
  member val Id: Guid = Guid.Empty with get, set
  member val Name: string = null with get, set
  member val Address: string = null with get, set
  member val BirthDate: DateOnly = DateOnly () with get, set

  new (name, address, birthDate) as this =
    Person ()
    then
      this.Name <- name
      this.Address <- address
      this.BirthDate <- birthDate

  with override this.ToString () =
        $"""Person {{ Id: {this.Id}, Name: {this.Name}, Address: {this.Address}, BirthDate: {this.BirthDate.ToString "dd.MM.yyyy"} }}"""

